src/pages/Contents/Sidebar/SidebarItems.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+export const SidebarItems = ({ isOpen }, { toggleSidebar }) => {
+  const [imageSrc, setImageSrc] = useState(
+    "/images/Contents/Icons/circle-check-regular.png"
+  );
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleCheck = () => {
+    if (isClicked) {
+      setImageSrc("/images/Contents/Icons/circle-check-regular.png");
+      setIsClicked(false);
+    } else {
+      setImageSrc("/images/Contents/Icons/circle-check-solid.png");
+      setIsClicked(true);
+    }
+  };
+
+  return (
+    <SidebarMenuItem isOpen={isOpen}>
+      <CheckIcon onClick={handleCheck} isOpen={isOpen} alt="v" src={imageSrc} />
```

Comment:
```suggestion javascript
 src={`/images/Contents/Icons/circle-check-${isClicked?"regular":"solid"}.png`} />
```
둘중에 하나 바뀌는 상황이라면, 이미지에서 바뀌는 스트링만 조건에 따라 바뀌게 할 수 있습니다.

Reply:
🪄 넵